.travis.yml

```diff
@@ -28,7 +28,6 @@ jobs:
   include:
   - stage: build
     before_install:
-    - ./.travis/find-qdrouterd-base-versions.sh
```

Comment:
This step saved us from having different router versions in the past. It should ensure that we don't have a different router image as a base for e.g. Dockerfile the "qdr configurator" is based on, or the router image we run.

I think we still need some kind of validation step to achieve that.

Reply:
Its a good point, because the check has already failed to detect that they are different :) 

https://github.com/EnMasseProject/enmasse/blob/master/iot/iot-proxy-configurator/Dockerfile#L6
https://github.com/EnMasseProject/enmasse/blob/master/Makefile.env.mk#L23

I think the approach we should take here is to make sure the ${ROUTER_IMAGE} env is used in the FROM for the proxy configurator Dockerfile. That should ensure everything comes from a single source.

I will make that modification in this PR if thats ok. 

(The original reason I remove the script, was that it barfed at the old OLM manifests)

Reply:
I've made the modification to templatize the iot-proxy-configurator Dockerfile so that the FROM gets set to the same value.